packages/plugins/src/utils.ts

```diff
@@ -15,8 +15,7 @@ export const createChugSplashRuntime = async (
   return {
     configPath,
     canonicalConfigPath,
-    remoteExecution:
-      process.env.FORCE_REMOTE_EXECUTION === 'true' ? true : remoteExecution,
+    remoteExecution,
```

Comment:
`FORCE_REMOTE_EXECUTION` is no longer required because we shouldn't be testing using the deploy task at all. We can probably now remove or simplify how the `remoteExecution` variable is used, but I chose not to go any further with it in this PR.